### ISSUE TITLE
Keyboard modifiers for partial throttle / brake input NOT working. RoR 0.4.7.0 win.

### ISSUE THREAD

**Issue by thomasatgit** [RigsOfRods/rigs-of-rods#1347]
_Hello everyone!_

Long story short: When I set input.map values:
```
TRUCK_ACCELERATE               Keyboard             SHIFT+W
TRUCK_ACCELERATE_MODIFIER_25   Keyboard             CTRL+W
TRUCK_ACCELERATE_MODIFIER_50   Keyboard             W
```
**I can't use acceleration unless the SHIFT is held.**
It should go 50% power when I only press W, 100% while holding W, then press and hold SHIFT as well.
25% when holding W, then press and hold CTRL as well.

EXPL+ tag sadly didn't help.

Long story: [My forum thread about this.](https://forum.rigsofrods.org/thread-1348-post-10846.html)

**Comment by Michael10055** [comment]
Are you sure it is not conflicting with the axle lock key?

```
TRUCK_TOGGLE_AXLE_LOCK         Keyboard             W 
```

**Comment by thomasatgit** [comment]
Sorry, I've never mentioned that I tried it as well with a vanilla, fresh install of RoR 0.4.7.0 win. When the throttle and brake keys ( and modifiers ) were altered to use the arrow keys the way I asked in the first post, it didn't work either.

Like this:
```
TRUCK_ACCELERATE               Keyboard             SHIFT+UP
TRUCK_ACCELERATE_MODIFIER_25   Keyboard             CTRL+UP
TRUCK_ACCELERATE_MODIFIER_50   Keyboard             UP
```
Same behaviour, I had to hold SHIFT to be able to accelerate with UP. Same goes for braking ( DOWN ).
